src/Avalonia.Animation/Animators/Animator`1.cs

```diff
@@ -89,6 +89,9 @@ protected T InterpolationHandler(double animationTime, T neutralValue)
             else
                 newValue = (T)lastKeyframe.Value;
 
+            if (lastKeyframe.KeySpline != null) // TODO: do we use firstKeyFrame or lastKeyframe?!
+                progress = lastKeyframe.KeySpline.GetSplineProgress(progress);
```

Comment:
This is correct, lastKeyframe should be used for the spline interpolation

Reply:
@Deadpikle  can you confirm that it matches WPF's behavior?

Reply:
OK, removed the TODO. Thanks for having me dig more for that. :)

Reply:
> P.S. Noticed that WPF allows for multiple KeyFrame on the same time. Avalonia probably needs to handle that, too.

Well, if the keyframes with the same Cue/KeyTime targets different properties then it should work. It doesnt make sense to allow 2 keyframes targeting the same prop with the same Cue/KeyTime imo.